utils/script_model.py

```diff
@@ -0,0 +1,32 @@
+import torch
+
+class ScriptModel(torch.nn.Module):
```

Comment:
Hey! Very curious to start using scripted models. Thanks for this addition. Could a test for the __init__ and forward() to validate these methods act as expected be relevant?

Reply:
yes, I added tests.